src/structures/MessageEmbed.js

```diff
@@ -65,7 +73,7 @@ class MessageEmbed {
      * The fields of this embed
      * @type {EmbedField[]}
      */
-    this.fields = data.fields ? this.constructor.normalizeFields(data.fields) : [];
+    this.fields = data.fields ? this.constructor._normalizeFields(this.skipValidation, data.fields) : [];
```

Comment:
Instead of making `normalizeFields` more or less a noop when skipping validation we could just revert to the previous behavior in those cases: 
https://github.com/discordjs/discord.js/blob/9c8aaf1bbcc220299e65cadb6ac54e1a7f208576/src/structures/MessageEmbed.js#L68

So, something like this:
```suggestion
    this.fields = [];
    if (data.fields) {
      this.fields = skipValidation ?
        data.fields.map(Util.cloneObject) :
        this.constructor.normalizeFields(data.fields);
    }
```

This would make all changes further down obsolete.
(And also keep this semver: patch instead of semver: minor)

Reply:
The reason I didn't want to just set `this.fields` equal to `data.fields` (more or less) is so that if `normalizeFields` got changed or expanded with more logic later on, this would still use it correctly whilst only skipping the validation part, not the whole method.

Does that make sense, or would it just be easier to set it directly and hope that no other logic gets added down the road?

Reply:
if more logic is added later down the road this can also be changed to use that logic

Reply:
`normalizeFields` already has additional logic in that it makes calls to `resolveString`, which using `cloneObject` won't. I stand by my decision that we should use `normalizeFields` with a flag to skip the error throwing validation.

Reply:
if you read what space suggested, the skipValidation parameter in the constructor still exists, and Util.cloneObject is used if that is true, any data discord sends is very likely to always be a string, so there is no need to call Util.resolveString on it

Reply:
if you look at https://github.com/discordjs/discord.js/issues/3892#issuecomment-594502208 you can see that the field causing the issue does have the `name` and `value` properties, which are both strings

Reply:
Yes, I'm aware of my own report and the issue it brings. That's not my point though -- this code had assumed that name & value will have contents. That was not the case and it broke. We should not then be making a fix that involves making other assumptions about the data, especially when there is already logic in place to handle situations that don't fit that assumption.

Reply:
I would strongly agree with space suggestion to fix this.